dockyard/common/container/scheduler/round_robin.py

```diff
@@ -0,0 +1,16 @@
+from oslo_config import cfg
+
+from dockyard.common.cluster import cluster
+from dockyard.common.container.scheduler.base import Scheduler
+
+
+class RoundRobinScheduler(Scheduler):
+    count = -1
```

Comment:
There should be some driver concept so that driver for etcd, consul etc can be used. And that is available in the configuration file.